http/api_router.go

```diff
@@ -21,16 +22,26 @@ type apiRouter struct {
 	tokenService model.TokenService
 }
 
+// Settings describe router's settings
+type Settings struct {
```

Comment:
I'm not sure it's the best way to pass settings to the Router. Please let's discuss better options for that.